src/components/App/App.spec.tsx

```diff
@@ -1,21 +1,115 @@
-import React, { Suspense } from 'react';
+import React from 'react';
 import { render } from '@testing-library/react';
-import { I18nextProvider } from '@wix/wix-i18n-config';
-import i18n from '../__mocks__/i18n';
 import App from './App';
+import RTLAppDriver from './AppDriver';
 
 describe('App', () => {
-  it('renders a title correctly', async () => {
-    const { findByTestId } = render(
-      <Suspense fallback="...loading">
-        <I18nextProvider i18n={i18n}>
-          <App />
-        </I18nextProvider>
-      </Suspense>,
+  let driver: RTLAppDriver;
+  const EMPTY = '';
+  const testData = {
+    firstName: 'Yair',
+    lastName: 'Dana',
+    color: 'Blue',
+  };
+
+  beforeEach(() => {
+    const { baseElement } = render(<App />);
+    driver = new RTLAppDriver(baseElement);
+  });
+
+  it('should display submitted info when user click submit', async () => {
+    await driver.when.enterFirstName(testData.firstName);
+    await driver.when.enterLastName(testData.lastName);
+    await driver.when.selectColor(testData.color);
+
+    await driver.when.submitButtonClick();
+
+    expect(await driver.get.savedDataFirstName()).toEqual(testData.firstName);
+    expect(await driver.get.savedDataLastName()).toEqual(testData.lastName);
+    expect(await driver.get.savedDataColor()).toEqual(testData.color);
+  });
+
+  it('should override save data when user submit another form', async () => {
+    const testOverrideData = {
+      firstName: 'Dummy First Name',
+      lastName: 'Dummy Last Name',
+      color: 'Red',
+    };
+
+    await driver.when.enterFirstName(testData.firstName);
+    await driver.when.enterLastName(testData.lastName);
+    await driver.when.selectColor(testData.color);
+
+    await driver.when.submitButtonClick();
+
+    await driver.when.enterFirstName(testOverrideData.firstName);
+    await driver.when.enterLastName(testOverrideData.lastName);
+    await driver.when.selectColor(testOverrideData.color);
+
+    await driver.when.submitButtonClick();
+
+    expect(await driver.get.savedDataFirstName()).toEqual(
+      testOverrideData.firstName,
     );
+    expect(await driver.get.savedDataLastName()).toEqual(
+      testOverrideData.lastName,
+    );
+    expect(await driver.get.savedDataColor()).toEqual(testOverrideData.color);
+  });
+
+  it('should clear form when user click clear', async () => {
+    await driver.when.enterFirstName(testData.firstName);
+    await driver.when.enterLastName(testData.lastName);
+    await driver.when.selectColor(testData.color);
+
+    await driver.when.clearButtonClick();
+
+    expect(await driver.get.firstNameValue()).toEqual(EMPTY);
```

Comment:
I understand you wanted to define that in a variable and it's a good practice for constants but maybe in the case of empty string it makes the tests more readable to have just ```suggestion
    expect(await driver.get.firstNameValue()).toEqual('');
```